yb-voyager/cmd/exportData.go

```diff
@@ -262,10 +265,25 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName) er
 	return nil
 }
 
+func filterTablePartitions(tableList []*sqlname.SourceName) []*sqlname.SourceName {
+	if source.DBType != POSTGRESQL || source.TableList != "" {
```

Comment:
Don't MySQL and Oracle also have partitioned tables?

Reply:
Yes they do, but the query used in `GetAllTableNames()` does return only parent table in case of oracle and mysql.
But in case of postgresql, table partitions are considered as normal tables and hence they are also returned in the list and hence we only need to filter them out for the postgresql

Reply:
makes sense